src/Data.h

```diff
@@ -184,6 +184,12 @@ struct DataFrame {
   bool TIMED_CUTDOWN_ENABLE = false;
   uint32_t TIMED_CUTDOWN_MILLIS = 0;
 
+  bool DEADMAN_ENABLED = true;
+  uint32_t DEADMAN_TIME = 50400000;
```

Comment:
not to be annoying but can u adda comment on how long this is? looks like 14hrs to be but would be good to have it obvious